### ISSUE TITLE
Fix misformatted project description in PyPI for version 2.24.0

### ISSUE THREAD

**Issue by impredicative** [psf/requests#5501]
The project description used to display fine at https://pypi.org/project/requests/2.23.0/. It doesn't at https://pypi.org/project/requests/2.24.0/. It is a reasonable expectation that it displays correctly.

**Comment by nateprewitt** [comment]
Hi @impredicative,

Thanks for the ticket. We're aware there were some issues with the long description upload this morning. Some of the PyPI infrastructure changed since our last upload and we ended up with incorrectly transformed markdown.

We're working on some larger changes to the README file itself and will upload a new one in a 2.24.1 patch. Current release time is TBD but we'll post an update here when it's done. I'll leave this open to track the issue for now.

**Comment by MartinThoma** [comment]
By the way, you could remove everything related to long_description from `setup.py` and instead put this in `setup.cfg`:

```
long_description = file: README.md
long_description_content_type = text/markdown
```

The Markdown of the README could also be simplified (essentially stripping the HTML). That part seems rather straight-forward. Should I make a PR?

**Comment by hugovk** [comment]
I did a test upload of `master` to TestPyPI and the README is now correctly rendered by PyPI:

* https://test.pypi.org/project/requests-readme-md-test/2.24.0/

**Comment by sethmlarson** [comment]
There are changes coming to the README to make sure it's pure Markdown as well.

**Comment by ssbarnea** [comment]
Is this going to be ever fixed as it looks really awful. IMHO, it would be much better UX to stick to pure text without any images or big advertisements, especially at the top of the page.

**Comment by nateprewitt** [comment]
This was resolved in v2.25.0.
